catalog/app/translations/en.json

```diff
@@ -1 +1,45 @@
-[]
+{
```

Comment:
1. messages, by convention, should be defined in `messages.js` file in component's directory (examples: [Auth/messages.js](https://github.com/quiltdata/quilt/blob/master/catalog/app/containers/Auth/messages.js), [usage](https://github.com/quiltdata/quilt/blob/master/catalog/app/containers/Auth/PassChange.js#L44))

2. i dont' think it makes any sense to put non-translatable things there (like syntax, for example)

3. i dont' think we should continue using the current approach to i18n, bc a) it makes things a little less simple and more indirect, requires some unnecessary boilerplate and adds little value (on the current stage of development); b) the intl lib we are using is outdated and needs to be upgraded

Reply:
I tried to set this implementation but lost as I dig further. I see two problems that I cannot resolve:
1. every message has two IDs (one is object's key, second is real id)
2. I don't understand where are real messages, besides "defaultMessage"s

---

Also I move syntax and example into data outside i18n

Reply:
I moved texts back into a component, removed `react-intl` use.